torch/_dynamo/variables/builder.py

```diff
@@ -1132,6 +1133,16 @@ def _clone_input(value):
             }
             assert "source" in options and options["source"] is not None
             kwargs["source"] = options["source"]
+
+            if not isinstance(example_value, torch._subclasses.fake_tensor.FakeTensor):
```

Comment:
This is a nitty thing, but I want to support running dynamo on programs involving fake tensors, so we shouldn't use this type of isinstance test to figure out if we are "internal" or "external" tensor